bin/cron/deploy_to_test

```diff
@@ -22,13 +22,17 @@ def main
 
   # TODO(asher): Add a reviewer (namely the next DOTD), add appropriate tags
   # (particularly deploy).
-  pr_number = GitHub.create_and_merge_pull_request(
+  pr_number = GitHub.find_or_create_pull_request(
     base: 'test',
     head: 'staging',
     title: 'DTT (Staging > Test) [robo-dtt]'
   )
   if pr_number.nil?
-    raise Exception.new('GitHub.create_and_merge_pull_request failed.')
+    raise Exception.new('GitHub.find_or_create_pull_request failed.')
```

Comment:
It might be helpful to on call engineers who will see this error to make this message more informative. Maybe it should be something like "Unable to find or create a Pull Request that merges the staging branch into the test branch."

Reply:
Is pr_number ever likely to be nil? Looking at the source code in Github.rb, I wouldn't expect `nil` to ever be returned.